flake.nix

```diff
@@ -0,0 +1,125 @@
+{
+  inputs = {
+    # dream2nix.url = "github:nix-community/dream2nix";
+    dream2nix.url = "github:blaggacao/dream2nix";
+  };
+
+  outputs = {
+    self,
+    dream2nix,
+    nixpkgs,
+  } @ inp: let
+    inherit (nixpkgs.legacyPackages.x86_64-linux) pkgs;
+    inherit (nixpkgs) lib;
+  in
+    (dream2nix.lib.makeFlakeOutputs {
+      systems = ["x86_64-linux"];
+      config.projectRoot = ./.;
+      config.packagesDir = "./locks";
+      source = ./.;
+      settings = [
+        {subsystemInfo.noDev = false;}
+      ];
+      packageOverrides = let
+        system = "x86_64-linux";
+        toplevel = "cardano-sdk";
+        subpackages = lib.filterAttrs (k: v: lib.hasPrefix "@cardano-sdk" k) self.packages.${system};
+        subpackagesNames = builtins.attrNames subpackages;
+        topDependencies =
+          builtins.removeAttrs
+          self.packages.${system}.${toplevel}.dependencies
+          (subpackagesNames ++ [toplevel]);
+      in
+        lib.recursiveUpdate (
+          builtins.mapAttrs (n: _: let
+            otherNames = lib.remove n subpackagesNames;
+            otherSubpackages = builtins.removeAttrs subpackages [n];
+          in {
+            # install-method.installMethod = "copy";
+            add-inputs.buildInputs = old: old ++ (lib.attrValues topDependencies) ++ [pkgs.nodejs-14_x.pkgs.yarn];
+            install-symlinks.postInstall = "ln -s $out/lib/node_modules/${n}/dist $out/dist";
+            add-type-notation.prePatch = lib.optionalString (n == "@cardano-sdk/core")
+            ''
+              substituteInPlace ./src/Cardano/util/subtractValueQuantities.ts \
```

Comment:
That's odd, why is this needed? We have many other relative imports from own package

Reply:
I have no good explanation. The TS compiler fails with the relative import claiming that it cannot infer xyz type. Without knowing for certain, it might be related to:

https://github.com/microsoft/TypeScript/issues/42873

(which does not contain qualified information)

Reply:
Seems related to needing to add those imports :thinking:

Reply:
Yes, indeed. Maybe we can figure this out and eradicate the root cause. :pray: